CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 # CHANGELOG
 
 ## 0.7.10dev
-* [Feature] Support flexible spacing `myvar=<<` operator ([#525](https://github.com/ploomber/jupysql/issues/525))
 
+* [Fix] Refactored `magic_cmd.py` by assigning each command logic to a separate file [#518]
```

Comment:
since this isn't user-facing, we can skip the changelog entry, I added the no-changelog entry label so the CI doesn't complain

Reply:
Got it!